### ISSUE TITLE
Debian 8 doesn't boot in GPT-FAT32 UEFI mode

### ISSUE THREAD

**Issue by pbatard** [pbatard/rufus#531]
Note: This is a __pure__ Debian issue, not a Rufus one.

Alas, like many others, the Debian guys have been drinking the ISOHybrid kool aid, and screwed all users who find it preferable to install their Linux distro from a filesystem __that can be easily edited from Windows__ (say, to force a specific video mode in case the default Debian one does not work, or add the WiFi firmware you need so that you'll be able to connect to the network post install), by making their EFI GRUB bootloader compatible with ISO9660 only, whereas it would have been _oh-so-frigging easy_ to add the FAT32 module as well...

That EFI GRUB bootloader is actually super weird: It includes a virtual `/` device containing only a fake `boot/grub/grub.cfg` with the following:
```
search --file --set=root /.disk/info
set prefix=($root)/boot/grub
source $prefix/x86_64-efi/grub.cfg
```

This would probably work well in practice, had the Debian guys NOT compiled their EFI GRUB with only ISO9660 filesystem support only (which certainly wasn't due to a space shortage), which means you can then forget about locating that `/.disk/info` on FAT32.

Distro maintainers, please heed my words: ISOHybrid IS a hack: It makes installations __more troublesome__ for the very users you are targeting (regular Joes who are trying to move away from Windows), prevents them from reusing their USB drives as they see fit, and, as with any hack, __will be__ more trouble than it's worth in the long run. Or just consider this: Wouldn't you be in arms if someone bastardized the ext4 filesystem to make it NTFS compatible, and then __forced__ everyone to use the bastardized version? Being able to edit your installation filesystem IS what you want. ISOHybrid prevents that!

**Comment by pbatard** [comment]
OK, looking more closely, it looks like the FAT module is included. Yet the drive is not being detected on a NUC in GPT/FAT32. Why?

**Comment by pbatard** [comment]
Submitted Debian bug [#789600](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=789600). Hopefully, they can fix this in their new ISO releases.

**Comment by pbatard** [comment]
Note: I was able to confirm that this issue is now fixed in Debian 8.3.
